### PR TITLE
Initial commit for remaining top menu updates

### PR DESCRIPTION
**Pull request by whalenda**
Remaining work for top menu

 X Bottom Spacing of Menu titles/Copy and images is not there. 
 X Font size is wrong. 20px desktop. 
 X Too much space between each section 37px current is a lot more
 X Images are 200x150 fine with current size, but the images are too spaced out.
 X Product section needs to be aligned with MCSWf logo it is not. Adjust leading of copy title and images to match figma.
 X Also the highlight should be a solid bar not just a line 3px 
 X Bottom of menu needs to be rounded (see figma) 
 X Menu section titles font size is wrong desktop 16px\
 X There is no RED line in the design/menu remove 
 X Mobile sub menu leading/i.e. Spacing between menus need to be more too close together see figma. 
 X Mobile close icon should be larger and aligned to the right .
 X On hover remove black background.

### CODE REVIEW

File: src/components/static/Navigation.svelte [hunk 1]
```diff
@@ -122,16 +122,16 @@
...
           {#each dropdownLinks as dropdown}
           <div
-            class="mt-4 transition transform hover:text-mcswf-gold hover:scale-110 px-14"
+            class="mt-4 transition transform hover:text-mcswf-gold hover:scale-110 pr-10"
             on:mouseleave={() => closeDropdown(dropdown.text)}
             on:mouseenter={() => openDropdown(dropdown.text)}
           >
             <a href={dropdown.url}>
-              <h2 class="font-bold">{dropdown.text}</h2>
+              <h2 style="font-size:20px" class="pb-1 font-bold">{dropdown.text}</h2>
```
Comment (wcrum) [inline src/components/static/Navigation.svelte]: @whalenda used `text-sm`, it is equivalent to 20px.
https://tailwindcss.com/docs/font-size

File: src/components/static/Navigation.svelte [hunk 2]
```diff
@@ -122,16 +122,16 @@
   </div>
     {#if !isMenuHidden}
       <!-- STANDARD NAVBAR -->
-      <div class="hidden mx-24 my-4 text-white lg:block xl:block 2xl:block h-96">
-        <div class="flex justify-center">
+      <div class="hidden text-white lg:block xl:block 2xl:block h-96" style="margin-left: 12.5%; margin-right: 12.5%;">
```
Comment (wcrum) [inline src/components/static/Navigation.svelte]: Probably should replace this with `mx-0.5`, its roughly equivalent.

https://tailwindcss.com/docs/margin

File: src/components/static/Navigation.svelte [hunk 3]
```diff
@@ -108,12 +108,12 @@
...
           aria-expanded="true"
           aria-controls="sidebar"
-          class="p-2 mr-2 text-white rounded-lg cursor-pointer lg:hidden hover:bg-mcswf-dark-blue hover:text-mcswf-gold"
+          class="text-white rounded-lg cursor-pointer lg:hidden hover:bg-mcswf-dark-blue hover:text-mcswf-gold"
         >
         {#if isMenuHidden}
         <Icon class="hover:text-mcswf-gold" icon="mdi:menu" />
         {:else}
-        <Icon class="hover:text-mcswf-gold" icon="mdi:close" />
+        <Icon height="20" width="20" class="hover:text-mcswf-gold" icon="mdi:close" />
```
Comment (wcrum) [inline src/components/static/Navigation.svelte]: Please change this to `class="h-5 w-5"`

Please refer to https://tailwindcss.com/docs/height

File: src/components/static/Navigation.svelte [hunk 4]
```diff
@@ -122,16 +122,16 @@
   </div>
     {#if !isMenuHidden}
       <!-- STANDARD NAVBAR -->
-      <div class="hidden mx-24 my-4 text-white lg:block xl:block 2xl:block h-96">
-        <div class="flex justify-center">
```
Comment (wcrum) [inline src/components/static/Navigation.svelte:126]: Removing `justify-center` makes it off centered, is this the expected behavior?

**Review by wcrum — CHANGES_REQUESTED** [review]
A couple small changes to use tailwind classes, but looks good to me.

Question regarding centering.

File: src/components/static/Navigation.svelte (hunk 2 above)
Comment (dfullaway) [inline src/components/static/Navigation.svelte]: xl and 2xl sections not needed. h-96 might also be a bit much, h-80 seems like it should be big enough.

File: src/components/static/Navigation.svelte
Comment (dfullaway) [inline src/components/static/Navigation.svelte:1]: When the page width is less than 1289 and larger than 1024, "Training and Career" wraps to another line, and the images are no longer aligned. I don't know that there is a way to fix this while also doing everything that Mas asked for.

File: src/components/static/Navigation.svelte (hunk 1 above)
Comment (whalenda) [reply src/components/static/Navigation.svelte]: updated to text-xl which is 20px

File: src/components/static/Navigation.svelte (hunk 4 above)
Comment (whalenda) [reply src/components/static/Navigation.svelte:126]: Updated this to justify-left to align it with "Product section needs to be aligned with MCSWf logo it is not."

File: src/components/static/Navigation.svelte (hunk 2 above)
Comment (whalenda) [reply src/components/static/Navigation.svelte]: Updated to keep in line, used slightly different tailwind classes

File: src/components/static/Navigation.svelte
Comment (scottpiercew) [reply src/components/static/Navigation.svelte:1]: Looks good! Added break points to change the font size and put images in a grid. When all the code is consolidated back into the main branch there is some final width alignment needed between the banner, navigation, nav dropdown, and page content.

**Pull request merged**